docs/API.md

```diff
@@ -506,3 +507,25 @@ push.clearAllNotifications(() => {
 	console.log('error');
 });
 ```
+
+## push.clearNotification(id, successHandler, errorHandler) - Android only
+
+Tells the OS to clear the notification that corresponds to the id argument, from the Notification Center
+
+### Parameters
+
+Parameter | Type | Default | Description
+--------- | ---- | ------- | -----------
+`id` | `number` | | The ID of the notification that will be cleared. |
+`successHandler` | `Function` | | Is called when the api successfully clears the notifications.
+`errorHandler` | `Function` | | Is called when the api encounters an error when attempting to clears the notifications.
+
+### Example
+
+```javascript
+push.clearNotification(145, () => {
```

Comment:
I wonder if it's worth to put the `id` argument last, to match the rest of the API. Although I think we got the order wrong on the rest, not here, but then to break it all would be a huge pain...

Reply:
I think that consistency is probably best here. Since the other methods are using `id` last, I will change this method to use it last as well. This way if in the future if the API is changed to use `id` first, this method will just be changed with the rest.